ext/gd/gd.c

```diff
@@ -2673,6 +2673,8 @@ static void php_imagepolygon(INTERNAL_FUNCTION_PARAMETERS, int filled)
 			RETURN_THROWS();
 		}
 		NPOINTS /= 2;
+	} else {
+		php_error_docref(NULL, E_DEPRECATED, "using the $num_points parameter is deprecated");
```

Comment:
```suggestion
		php_error_docref(NULL, E_DEPRECATED, "Using the $num_points parameter is deprecated");
```

Just like in https://github.com/php/php-src/pull/6777#discussion_r596105939 :D